package.json

```diff
@@ -52,9 +52,9 @@
     "vscode-test-utils": "^0.0.8"
   },
   "scripts": {
-    "precompile": "node ./node_modules/vscode/bin/install",
-    "compile": "tsc",
-    "lint": "tslint --project tsconfig.json",
+    "postinstall": "node ./node_modules/vscode/bin/install",
+    "compile": "tsc -p ./",
```

Comment:
`-p ./` is implied when there is a `tsconfig.json` at project root, so why are you doing this?